crates/anstyle/src/color.rs

```diff
@@ -598,6 +598,7 @@ impl core::fmt::Display for DisplayBuffer {
 }
 
 #[cfg(test)]
+#[cfg(feature = "std")]
```

Comment:
Could you instead put this on the test in question?  If we add more unit tests that want to test core functionality, they should still be able to run

Reply:
I would have, but that produced a warning about the `use super::*;` being unused.

I can still do it if you want?

Reply:
Eh, oh well. Something will have to change when adding tests in the future.